Matcher import path is broken in get-queries-for-element typings file

The Matcher import in `typings/get-queries-for-element.d.ts` is incorrect.

* `dom-testing-library` version: 2.3.1
* `react` version: 16.3.1
* `node` version: 8.11.1
* `npm` (or `yarn`) version: 6.0.1

### Relevant code or config:

The current (incorrect) code is:
```js
import {Matcher} from 'matches'
```

The correct import is as follows:

```js
import {Matcher} from './matches'
```

### What you did:
Attempt to call `ReactTesting.renderIntoDocument(...)` in a `foo.test.tsx` file fails.
Unfortunately the code is in a private GitHub repo that can't be shared here.


### What happened:
TypeScript compilation of the app fails and returns the following error message:

```
node_modules/dom-testing-library/typings/get-queries-for-element.d.ts
(1,23): Cannot find module 'matches'.
```

--- a/typings/get-queries-for-element.d.ts
+++ b/typings/get-queries-for-element.d.ts
@@ -1,4 +1,4 @@
-import {Matcher} from 'matches'
+import {Matcher} from './matches'
 import * as queries from './queries'
 
 export type BoundFunction<T> = T extends (

